content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+
+* Mesh Operator, who manages the service mesh control plane installation and upgrade.
+
+* Mesh Admin, often referred as Platform Owner, who owns the service mesh platform and defines the overall strategy and implementation for service owners to adopt service mesh.
+
+* Mesh User, often referred as Service Owner, who owns one or more services in the mesh.
+
+Istio currently requires the control plane to run in one of the clusters in the mesh, leading to a lack of separation between the mesh operator and the mesh admin. We're introducing an alternative deployment model for Istiod which enables mesh operators to install and manage the mesh control plane on separate clusters. This new deployment model allows a clear separation between mesh operators and mesh admins. With this new model, Istio mesh operators can run Istio control plane for mesh admins while mesh admins can still control the configuration of the control plane without worrying about installing or managing the Istio control plane. This model is transparent to mesh users.
+
+## External Control Plane Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you will have an Istiod control plane installed in a single cluster like the diagram below:
+
+{{< image width="100%"
+    link="single-cluster.svg"
+    alt="Istio mesh in a single cluster"
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+With this new deployment model in Istio 1.7, it is possible to run Istiod on a separate cluster (the `external control plane cluster`) from services, as shown in the diagram below. The `external control plane cluster` is owned by mesh operator while the mesh admin owns the cluster that has services deployed in the mesh. Mesh admin has no access to the `external control plane cluster`. For mesh operators, you can follow the [external istiod single cluster step by step guide](https://github.com/istio/istio/wiki/External-Istiod-single-cluster-steps) to explore more on this. Note: In some internal discussions among Istio maintainers this model was known as "central istiod".
+
+{{< image width="100%"
+    link="single-cluster-external-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in an external control plane cluster"
+    >}}
+
+Mesh admins can expand the service mesh to multiple clusters, which are managed by the same Istiod running in the `external control plane cluster`. The `config cluster` also serves as the Istio configuration cluster, in addition to running services. The `external control plane cluster` reads Istio configurations from the `config cluster` and Istiod pushes configuration to the data plane running in both the `config cluster` and `remote cluster` per the diagram below.
```

Comment:
```suggestion
Mesh admins can expand the service mesh to multiple clusters, which are managed by the same Istiod running in the external cluster. None of the mesh clusters are {{< gloss "primary cluster" >}}primary clusters{{< /gloss >}}, in this case. They are all {{< gloss "remote cluster" >}}remote clusters{{< /gloss >}}. However, one of them also serves as the Istio configuration cluster, in addition to running services. The external control plane reads Istio configurations from the `config cluster` and Istiod pushes configuration to the data plane running in both the config cluster and other remote clusters as shown in the diagram below.
```

Reply:
That is another reason I don't think the term "primary cluster" is suitable.  You can have none of them! 🤨

Reply:
Maybe not ideal, but "primary cluster" seems OK to me. A standalone mesh needs at least one primary cluster in it. The alternative is to use an external control plane to control the mesh. We had a long discussion before agreeing on the name. One other reasonable suggestion, "control cluster", was rejected for some reason I don't remember. @nmittler @smawson Do you remember?

Reply:
I recall preferring `control cluster` specifically because `primary` implies `secondary`, which is commonly used more in failover scenarios. @smawson do u recall?